dataset/dictionary/y-events.json5

```diff
@@ -242,72 +249,83 @@
   //   tags: [ "event", "object" ],
   // },
   {
-    en: "water cannon",
+    en: "Water Cannon",
     ja: "ウォーターキャノン",
+    // zhCN: "水炮",
```

Comment:
Can you add a comment to explain why this is commented out?

Reply:
I was ambiguous when I added the Chinese translation, and I just made a second confirmation